### ISSUE TITLE
Minor bug/cosmetic : StaxRip tab character rendering slightly different than CMD causing misalignment

### ISSUE THREAD

**Issue by BillyBob1354** [staxrip/staxrip#1243]
Seen in SVT-AV1 with "film grain synth" is enabled.

StaxRip (tabs):
![ZZZ Stax](https://github.com/staxrip/staxrip/assets/46253676/819ae74e-6857-40ec-805a-d8f803652b0d)

CMD (converts tabs to spaces):
![ZZZ CMD](https://github.com/staxrip/staxrip/assets/46253676/3e096864-8fd8-4ee6-9f41-a84d66de639e)

Notepad (tabs):
![ZZZ Notepad](https://github.com/staxrip/staxrip/assets/46253676/f77a7b57-237e-43cf-bd5d-770966f98b3b)

**Comment by Dendraspis** [comment]
As StaxRip just passes the output through, I think that the output has to be adjusted.

When it comes to the code lines, I put them to `Notepad++` and this is what came out after replacing `\t` with tabs:
<img src=https://github.com/staxrip/staxrip/assets/50659978/26e5c585-6eed-42ce-a358-0e56acbc3dc4 width=600>

Here on GitHub the result looks like:
```
SVT [config]: preset / tune / pred struct 					:
SVT [config]: BRC mode / target bitrate (kbps)				:
SVT [config]: BRC mode / rate factor 					:
SVT [config]: film grain synth / denoising / level 				:
```

And this is how it looks for me:
![image](https://github.com/staxrip/staxrip/assets/50659978/f64b178a-fbc3-4577-90f9-4a660dfded5d)

**Comment by BillyBob1354** [comment]
The command prompt's tab stops are 8 characters wide.
Notepad++ default is 4.

@Dendraspis and @BillyBob1354's screenshots of the StaxRip output show that its tab width is not the same between the computers (likely caused by the monitor scale setting in windows or just a different font size). StaxRip seems to be using something other than character width for tabs like pixel width or something else which causes the tab width variation.

Changing StaxRip's rendering of tabs to 8 characters wide (CMD Default) should fix the alignment issue.

**Tab size 1-8 Notepad++**
![Tab stop](https://github.com/staxrip/staxrip/assets/46253676/bb00f2a2-ed06-4924-b45d-7dada2feeb04)

**Comment by Dendraspis** [comment]
StaxRip does not render anything, it takes the raw output from the process.

We will wait till Patman86 updates the encoder....

**Comment by BillyBob1354** [comment]
https://stackoverflow.com/questions/22888999/find-out-default-tab-size-in-c-sharp-richtextbox

> The default size in pixels for a tab in a RichTextBox is apparently 48 pixels, regardless of font or font size. This is set as default by .NET without me touching the SelectionTabs array.

I tested and StaxRip has a fixed tab width of 48 pixels. The misalignment is caused by how StaxRip displays the "raw output" of SVT-AV1, it expects 8 character wide tabs not the 48 pixel wide tabs of StaxRip.

Scale Factor: 1.00 - font: Default
![Screenshot 2024-02-23 160737](https://github.com/staxrip/staxrip/assets/46253676/ac5a735a-b7a5-41da-9664-3d0e75f03280)

Scale Factor: 1.50 - font: Default
![Screenshot 2024-02-23 160742](https://github.com/staxrip/staxrip/assets/46253676/fae80b52-6379-4c1b-a253-6a3ad1134a2d)

Scale Factor: 2.00 - font: Source Code Pro Black
![Screenshot 2024-02-23 165834](https://github.com/staxrip/staxrip/assets/46253676/fcafb329-3b54-4c68-adcb-ba5277e0a029)

**Two solutions:**
1. **"Fix StaxRip"** - display tabs 8 characters wide like the windows command prompt.
Any program that expects the default tab width of the windows command prompt will be fixed.

2. **"Fix SVT-AV1-Enc app"** - convert text output to spaces internally, just use spaces. (do not use tabs)
Seems best to avoid tabs because the inconsistent default widths.
The program output will line up in anything that using a monospaced font; CMD, StaxRip, Notepad++, github. (all these have different default tab widths)
All the other default StaxRip encoders that I checked don't use tabs in there output, avoiding the problem with tabs.

Seems you were going for "solution 2" and it seems to be the better solution for SVT-AV1, but I would still like to see "solution 1" too because if any external program is used that still outputs tabs (probably expects the 8 character wide tabs of CMD) you can get nice alignment.

**Comment by Dendraspis** [comment]
A fixed tab size in pixels - .NET at its best.  👏😂  😵‍💫  🤦‍♂️ 

🤔   Yeah, it would be nice to have something like this:

![image](https://github.com/staxrip/staxrip/assets/50659978/893e86d2-04af-4efb-9f56-21d2544d92f4)

If only we could find someone who could implement that.....  😏 

The problem is, that it is also pretty time consuming and on top of `Output Highlighting` it makes a difference. So Patman86's modification is still important and he already confirmed to change it to the system he uses in his `x265` mod.

**Comment by Dendraspis** [comment]
`SvtAv1EncApp` should be fine now with the latest version included in `v2.37`.

Additionally there will be an adjustment, that replaces tabs with spaces in `v2.37.3` for the whole Processing window.
